README.md

```diff
@@ -170,13 +170,51 @@ Option 1: If `NHAIS_MONGO_HOST` is defined then the adaptor forms a connection s
 | NHAIS_MONGO_TTL                  | P30D    | (Optional) Time-to-live value for inbound and outbound state collection documents as an [ISO 8601 Duration](https://en.wikipedia.org/wiki/ISO_8601#Durations)
 | NHAIS_COSMOS_DB_ENABLED          | false   | (Optional) If true the adaptor will enable features and workarounds to support Azure Cosmos DB
 
-Option 2: If `NHAIS_MONGO_HOST` is undefined then the the adaptor uses the connection string provided:
+Option 2: If `NHAIS_MONGO_HOST` is undefined then the adaptor uses the connection string provided:
 
 | Environment Variable             | Default                   | Description 
 | ---------------------------------|---------------------------|-------------
 | NHAIS_MONGO_DATABASE_NAME        | nhais                     | Database name for Mongo
 | NHAIS_MONGO_URI                  | mongodb://localhost:27017 | Mongodb connection string
 
+###AWS DocumentDB TLS configuration
+
+AWS DocumentDB uses a private CA and thus requires managing a custom keystore for the CA certificates.
+
+To use TLS it has to be enabled in DocumentDB instance and Mongo connection string has to contain `tls=true` parameter
+(see [TLS options for Mongo connection string](https://docs.mongodb.com/manual/reference/connection-string/#tls-options) for details)
```

Comment:
To use TLS, it has to be enabled in the DocumentDB instance with the Mongo connection string containing the `tls=true`.  Additionally, the environment variable `NHAIS_MONGO_DOCUMENTDB_TLS_ENABLED` must be set to `true`.  For more information on TLS configuration in MongoDB see [TLS options for Mongo connection string](https://docs.mongodb.com/manual/reference/connection-string/#tls-options).